tasks/fix-cat2.yml

```diff
@@ -1,25 +1,21 @@
 ---
-- name: "MEDIUM | RHEL-07-010030 | PATCH | The Red Hat Enterprise Linux operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a graphical user logon."
-  command: "true"
-  changed_when: no
+### RHEL-07-010030 | RHEL-07-010040 combined as related tasks in regards to a config file no other content will be in.
+- name: "MEDIUM | RHEL-07-010030 | RHEL-07-010040 | PATCH | The Red Hat Enterprise Linux operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a graphical user logon."
+  template:
```

Comment:
```suggestion
  copy:
```

Reply:
why are we nitpicking this?

Reply:
When maintaining the role it's a pain to have to open yet another file to see what's being done, when that file is literally 3 lines.

Reply:
I'm ok with using template over copy here. I actually prefer to have it separated out. It is not that big a deal to reference a template file and if a change needs to be made in the template it limits the commit blast radius to the template file and not the already extremely long `fix-cat2.yml` file. I mean the file is 2500 lines as is, minimizing additional lines does more for readability and maintainability then using copy here.